packages/manifold/contracts/gachaclaims/ISerendipityLazyClaim.sol

```diff
@@ -39,9 +39,9 @@ interface IGachaLazyClaim {
   error CannotMintMoreThanReserved();
   error CannotMintFromContract();
 
-  event GachaClaimInitialized(address indexed creatorContract, uint256 indexed instanceId, address initializer);
-  event GachaClaimUpdated(address indexed creatorContract, uint256 indexed instanceId);
-  event GachaClaimMintReserved(
+  event SerendipityClaimInitialized(address indexed creatorContract, uint256 indexed instanceId, address initializer);
+  event SerendipityClaimUpdated(address indexed creatorContract, uint256 indexed instanceId);
+  event SerendipityClaimMintReserved(
```

Comment:
renaming these events -- will need to update scanner + lambda code (possibly) with new events to listen to once deployed